ReactWindows/ReactNative/Views/Image/Composition/EffectManager.cs

```diff
@@ -0,0 +1,130 @@
+            element.SizeChanged += (sender, e) =>
+            {
+                foreach (var child in _containerVisual.Children)
+                {
+                    child.Size = new Vector2((float)e.NewSize.Width, (float)e.NewSize.Height);
+                }
+            };
+        }
+
+        /// <summary>
+        /// Set the blur radius of the image view.
+        /// </summary>
+        /// <param name="blurRadius">The blur radius value.</param>
+        public void SetBlurRadius(double blurRadius)
+        {
+            ApplyEffect(BlurEffect.Get((float)blurRadius), BlurEffect.Name);
+        }
+
+        /// <summary>
+        /// Set the tint color of the image view.
+        /// </summary>
+        /// <param name="color">The tint color value.</param>
+        public void SetTintColor(uint color)
+        {
+            // Change the composite mode of the root visual to DestinationInvert.
+            // This is only needed for the tint color effect.
+            _rootVisual.CompositeMode = CompositionCompositeMode.DestinationInvert;
+
+            // Create the tint effect.
+            ApplyEffect(TintColorEffect.Get(color), TintColorEffect.Name);
+        }
+
+        private SpriteVisual CreateSpriteVisual(string name, CompositionBrush brush)
+        {
+            var spriteVisual = _containerVisual.Children.FirstOrDefault(
+                child => child.Comment.Equals(name)) as SpriteVisual;
+
+            if (spriteVisual != null)
+            {
+                var oldBrush = spriteVisual.Brush;
+                spriteVisual.Brush = brush;
+                oldBrush.Dispose();
+            }
+            else
+            {
+                spriteVisual = _compositor.CreateSpriteVisual();
+                spriteVisual.Brush = brush;
+            }
+
+            spriteVisual.Comment = name;
+            return spriteVisual;
+        }
+
+        private void ApplyEffect(IGraphicsEffect effect, string name)
+        {
+            using (effect as IDisposable)
+            using (var effectFactory = _compositor.CreateEffectFactory(effect, null))
+            {
+                // Create the effect brush
+                var effectBrush = effectFactory.CreateBrush();
+                var backdropBrush = _compositor.CreateBackdropBrush();
+                effectBrush.SetSourceParameter("Source", backdropBrush);
+
+                // Create the sprite visual and update its brush
+                var spriteVisual = CreateSpriteVisual(name, effectBrush);
+
+                // Insert the sprite visual to the container visual
+                _containerVisual.Children.InsertAtTop(spriteVisual);
+            }
+        }
+
+        /// <summary>
+        /// Disposes the Windows.UI.Composition resources.
+        /// </summary>
+        protected virtual void Dispose(bool disposing)
+        {
+            if (disposing)
+            {
+                _containerVisual.Dispose();
+            }
+        }
```

Comment:
This case you may need to seal the EffectManager class to delete the Dispose(bool) pattern, since its internal, not private.